source/delayed_capture/index.html.md.erb

```diff
@@ -18,27 +18,34 @@ You cannot use delayed capture for:
 * payments made with [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
 * [recurring payments](/recurring_payments)
 
-To use this feature, include `"delayed_capture": true` in the body of a [Create new payment](/making_payments/#creating-a-payment) request.
+To use this feature, include `"delayed_capture": true` in the body of your [request to create a new payment](/making_payments/#creating-a-payment):
+
+`POST /v1/payments`
+
+```json
+{
+  "amount": 5000,
+  "reference" : "LE2504",
+  "description": "Pay a penalty charge notice (PCN)",
+  "return_url": "https://your.service.gov.uk/completed",
+  "delayed_capture": true
+}
+```
 
-The user experience matches the current payment flow. Once your user selects
-__Confirm__ on the __Confirm your details__ page:
+Your user will follow the standard payment flow. After your user selects __Confirm__ on the __Confirm your details__ page:
 
-- the [payment status](/api_reference/#payment-status-lifecycle) will change to `capturable`
-- your service can make a request to the
-`POST /v1/payments/{PAYMENT_ID}/capture` endpoint to send a delayed capture
-request
+- the [payment's status](/api_reference/#payment-status-lifecycle) will change to `capturable`
+- your service can make a request to the `POST /v1/payments/{PAYMENT_ID}/capture` endpoint to send a delayed capture request
```

Comment:
I am being super interrogative here because obviously I only have this one journey to follow in my head, but if your service *can* make a request, what governs if it does or doesn't - is it predicated on whether there's a delayed capture to send? This is more me just asking this than definitely saying it's not clear...

Reply:
So this is actually a really confusingly worded line that I've never really considered before!

For a standard payment, a service sends an API request to create a payment. They send their user to a URL we provide. The user enters their payment details. A payment service provider takes the money from the user's bank account.

For a delayed capture payment, a service sends an API request to create a payment and includes `"delayed_captured": true` in the request. They send their user to a URL we provide. The user enters their payment details. We hold their payment details until the service tells us to  take the payment by making a request to our API. We then tell the payment service provider to take the money.

So to answer your question, yes, it is predicated on whether there's a delayed capture to send. This endpoint only works if the payment specified by the `{PAYMENT_ID}` was created with the `"delayed_capture": true` parameter included. 

Whether they send the capture request or not depends on why they were delaying capture in the first place. For example, some services will make a user enter their payment details, then the service will perform some checks on the user (identity, credit, whatever). If the user fails those checks, the service may not send the payment capture request.

All this to say, I think this it might be a bit clearer if I add this to the bullet point:

> - your service can make a request to the `POST /v1/payments/{PAYMENT_ID}/capture` endpoint when you want to capture the delayed payment

Reply:
I hate to just go "Yep, sounds good" after such a detailed explanation, but I think you've nailed it. Because now I understand that capturing the delayed payment is an "option" depending on a thing that happens, and you've captured that now. Yay!